index.html

```diff
@@ -0,0 +1,68 @@
+          <label for="email" id="email-label">Email</label><br>
+            <input id="email" type="email" placeholder="Your email" required><br>
+          <label for="number" id="number-label">number</label><br>
+            <input id="number" type="number" min="5" max='20' placeholder="Your age" required><br>
+          <p class="description">Which option best describes your course role?</p>
+              <!-- <input id="course" type="dropdown" placeholder="choosen course" required> -->
+          <select class="dropdown" name="role" required> 
+            <option  disabled selected value > select your course</option>
+            <option value="course">mathematics</option>
+            <option value="course">English</option>
+            <option value="cou">Chemistry</option>
+            <option value="course">Yoruba</option>
+            <option value="cou">Physics</option>
+            <option value="course">Biology</option>
+          </select>
+          <p class="description">What is your best course ?</p>
+              <!-- <input id="course" type="dropdown" placeholder="choosen course" required> -->
+          <select id="dropdown" name="role" required> 
+            <option  disabled selected value > select your course</option>
+            <option value="course">mathematics</option>
+            <option value="course">English</option>
+            <option value="cou">Chemistry</option>
+            <option value="course">Yoruba</option>
+            <option value="cou">Physics</option>
+            <option value="course">Biology</option>
+            </select><br>
+          <p>Recommendation</p>
+          <div class="radio">
+            <label for="yes"><input id="yes" type="radio" name="yes" value="loving">Yes</label><br>
+            <label for="no"><input id="no" type="radio" name="no" value="loving">No</label><br>
+            <label for="maybe"><input id="maybe" type="radio" name="maybe" value="loving">Maybe</label><br></div>
+            <p class="description">What would you like to see improved?</p>
+            <label for="frontend"><input id="frontend" type="checkbox" name="frontend" value="loving">Front-end project</label><br>
+            <label for="backend"><input id="backend" type="checkbox" name="backend" value="loving">Back-end project</label><br>
+            <label for="fullstack"><input id="fullstack" type="checkbox" name="fullstack" value="loving">Fullstack project</label><br>
+            <label for="devop"><input id="devop" type="checkbox" name="devop" value="loving">DEVop project</label><br>
+            <label for="engineer"><input id="engineer" type="checkbox" name="engineer" value="loving">VR engineer project</label><br>
+            <label for="computerscience"><input id="computerscience" type="checkbox" name="computerscience" value="loving">computer-cience project</label><br>
+            <label for="uiux"><input id="uiux" type="checkbox" name="uiux" value="loving">UI/UX project</label><br>
```

Comment:
Not have wrapped inside `label`

Reply:
- Here we can say politely something like:
  * It would be nice to use the correct syntax in line 49. You can see an example here [label](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)